packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -1904,3 +1904,102 @@ let rows = [
   </TableBody>
 </TableView>
 ```
+
+## Expandable Rows <VersionBadge version="alpha" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
+
+By default, TableView doesn't allow expandable rows, but this can be enabled using the `hasExpandableRows` prop.
+Use `defaultExpandedKeys` to provide a default set of expanded rows. Note that the value of the expanded keys must match the `key` prop of the Row.
+
+You'll first need to enable a feature flag in your app to use this feature.
+
+```tsx example
+import {enableTableNestedRows} from '@react-stately/flags';
+
+enableTableNestedRows();
+```
+
+The example below enables expandable rows, and uses `defaultSelectedKeys` to expand rows with keys "1" and "2".
+
+```tsx example
+<TableView aria-label="Example table with expandable rows" hasExpandableRows defaultExpandedKeys={['1', '2']} height="size-3000">
+  <TableHeader>
+    <Column>Name</Column>
+    <Column>Capital</Column>
+    <Column align="end">Population</Column>
+  </TableHeader>
+  <TableBody>
+    <Row key="1">
+      <Cell>United States</Cell>
+      <Cell>Washington, D.C.</Cell>
+      <Cell>331.9 million</Cell>
+      <Row key="2">
+        <Cell>California</Cell>
+        <Cell>Sacramento</Cell>
+        <Cell>39.24 million</Cell>
+        <Row key="3">
+          <Cell>San Jose</Cell>
+          <Cell>-</Cell>
+          <Cell>983.4 thousand</Cell>
+        </Row>
+      </Row>
+    <Row key="4">
+      <Cell>Texas</Cell>
+      <Cell>Austin</Cell>
+      <Cell>29.53 million</Cell>
+    </Row>
```

Comment:
Mega nit, I think it might be useful having a child row for Texas just to highlight that only a subset of keys have been specified for expansion.